src/callBuilder.js

```diff
@@ -0,0 +1,139 @@
+import {NotFoundError, NetworkError, BadRequestError} from "./errors";
+
+let URI = require("URIjs");
+let URITemplate = require('URIjs/src/URITemplate');
```

Comment:
Peeking at the code for URIjs, it seems like this export is also available from the root export.  For example:
```js
let URITemplate = require("URIjs").URITemplate
```

If that works, we should use it over directly requiring into the `src` structure of the project.  In general, we shouldn't assume the source layout is part of the packages external contract, and it could therefore break without notice.